### PR TITLE
Added test for GET request

### PR DESCRIPTION
**Pull request by rajkumardongre**

### CODE REVIEW

**Review by milancurcic — CHANGES_REQUESTED** [review]
Thanks @rajkumardongre.

1. Error messages should be written to stderr. In Fortran that's accessible as

```fortran
use iso_fortran_env, only: stderr => error_unit
...
write(stderr, *) "error message"
```

When you write with `print *, ...` by default it goes to standard output.

2. If there are failures in the test program, you should stop with a non-zero code, i.e. `error stop 1`. This will signal to external tools like `fpm`, `CMake` or shell that the program did not complete successfully.

**Comment by rajkumardongre** [comment]
Thanks @milancurcic, i have made the required changes

File: test/test_get.f90 [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    end do
+
+    if (header_counter /= original_header_count) then
+        ok = .false.
+        msg = msg // 'test case 4, '
+    end if
+
+    if (.not. ok) then
+        msg = msg // 'Failed.'
+        write(stderr, *) msg
```
Comment (milancurcic) [inline test/test_get.f90:58]:
```suggestion
        write(stderr, *) msg
        error stop 1
```

**Comment by rajkumardongre** [comment]
Should I assume that when I use the `write(stderr, *) "error message"` statement to display an error message, I also need to include an `error stop 1` statement to stop the program execution when the error occurs?

**Comment by milancurcic** [comment]
Yes, when you `write(stderr, *) "error message"`, it merely prints it to standard error stream, but the program can still complete with a 0 status, so we need `error stop 1`.

**Review by milancurcic — APPROVED** [review]

**Pull request merged**